src/pages/BitcoinDashboardV2/components/Tabs/Tabs.tsx

```
@@ -0,0 +1,63 @@
+  },
+  {
+    key: KeysFilter.checkpoint,
+    text: 'Checkpoint'
+  },
+  {
+    key: KeysFilter.convert_bitcoin_v2,
+    text: 'Convert Bitcoin V2'
+  }
+];
+
+export const Tabs: FC<{}> = ({}) => {
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)